src/freenet/clients/http/PproxyToadlet.java

```diff
@@ -134,12 +134,11 @@ public void handleMethodPOST(URI uri, final HTTPRequest request, ToadletContext
 			
 			if (request.isPartSet("submit-official")) {
 				final String pluginName = request.getPartAsStringFailsafe("plugin-name", 40);
-				final String pluginSource = request.getPartAsStringFailsafe("pluginSource", 10);
-				
+
 				node.executor.execute(new Runnable() {
 					@Override
 					public void run() {
-						pm.startPluginOfficial(pluginName, true, true, "https".equals(pluginSource));
```

Comment:
I fear that startPluginOfficial is (used as) public API — can you add a deprecated compatibility method that just ignores the two force options and calls the real one without them?